activesupport/lib/active_support/cache/file_store.rb

```diff
@@ -21,6 +21,15 @@ class FileStore < Store
       GITKEEP_FILES = [".gitkeep", ".keep"].freeze
 
       def initialize(cache_path, options = nil)
+        if cache_path == {}
```

Comment:
[`lookup_store` calls `extract_options`](https://github.com/rails/rails/blob/291a3d2ef29a3842d1156ada7526f4ee60dd2b59/activesupport/lib/active_support/cache.rb#L65) which returns `{}` if no options are provided.

the cache store needs to work even if the path doesn't exist yet, which is why i can't just check `Dir.exist(cache_path)` or something like that here.